lib/nostrum/struct/application_command_interaction_data_option.ex

```diff
@@ -0,0 +1,94 @@
+defmodule Nostrum.Struct.ApplicationCommandInteractionDataOption do
+  @moduledoc "Struct for command invocation arguments."
+
+  # seasons greetings from `AbstractBeanModelParserFactory.java`.
```

Comment:
This is very well written! Consider a career in literature if the whole programming thing doesn't work out. 

I think we both know this should probably be removed, but honestly I don't hate it. Maybe we meet half way and just keep the first paragraph? 😸

Reply:
I dropped it: 63eca89